File: Java/Produk.java
public class Produk {
    private String kodeProduk;
    private String namaProduk;
    private int harga;

    public Produk(String kodeProduk, String namaProduk, int harga) {
        this.kodeProduk = kodeProduk;
        this.namaProduk = namaProduk;
        this.harga = harga;
    }

    public String getKodeProduk() { return kodeProduk; }
    public String getNamaProduk() { return namaProduk; }
    public int getHarga() { return harga; }

    public void setKodeProduk(String kodeProduk) { this.kodeProduk = kodeProduk; }
    public void setNamaProduk(String namaProduk) { this.namaProduk = namaProduk; }
    public void setHarga(int harga) { this.harga = harga; }
}

File: Python/Produk.py
class Produk:
    def __init__(self, kodeProduk, namaProduk, harga):
        self.kodeProduk = kodeProduk
        self.namaProduk = namaProduk
        self.harga = harga

    def getKodeProduk(self):
        return self.kodeProduk

    def getNamaProduk(self):
        return self.namaProduk

    def getHarga(self):
        return self.harga

    def setKodeProduk(self, kodeProduk):
        self.kodeProduk = kodeProduk

    def setNamaProduk(self, namaProduk):
        self.namaProduk = namaProduk

    def setHarga(self, harga):
        self.harga = harga
